style.css

```diff
@@ -0,0 +1,169 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+html * {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  overflow-x: hidden;
+}
+
+.crete-font {
+  font-family: "Crete Round", serif;
+}
+
+.inter-font {
+  font-family: "Inter", sans-serif;
+}
+
+.image-container img {
+  width: 100%;
+  height: 100%;
+}
+
+ul {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+.d-grid {
+  display: grid;
+}
+
+.d-flex {
+  display: flex;
+}
+
+.justify-center {
+  justify-content: center;
+}
+
+.align-center {
+  align-items: center;
+}
+
+.sr-only {
+  display: none;
+}
+
+.list li {
+  float: left;
+}
+
+/* Start of header styles */
+.mobile-container {
+  max-width: 480px;
+  margin: auto;
+  background-color: #555;
+  height: 500px;
+  color: white;
+  border-radius: 10px;
+}
+
+.topnav {
+  overflow: hidden;
+  background-color: #555;
+  position: relative;
+  width: 327px;
+}
+
+.topnav #myLinks {
+  display: none;
+}
+
+.topnav a {
+  color: white;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+  display: block;
+}
+
+.topnav a.icon {
+  background: #555;
+  display: block;
+  position: absolute;
+  right: 0;
+  top: 0;
+}
+
+.topnav a:hover {
+  color: rgb(245, 221, 6);
+}
+
+/* Start Navbar */
+
+/* Start Headline */
+
+section.headline {
+  position: relative;
+  background-color: rgba(28, 26, 25, 1);
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  background-repeat: no-repeat;
+  padding: 120px 0 10px 24px;
+  height: 590px;
+  background-image: url(./imgs/small.png);
+  background-position: 102% 100%;
+  width: 327px;
```

Comment:
- [OPTIONAL] Please don't use fixed-width use percentage or media-queries in order to make your project responsive for the different mobile devices.
![Captura de pantalla_2021-09-21_18-49-40](https://user-images.githubusercontent.com/53111464/134262254-93dea07d-62e0-4606-b47c-9153cbdf2689.png)
**Mobile device: pixel2
Screen width: 411px**